### DISCUSSION TITLE
New diagram causing crash on 17.4.2 Windows

### DISCUSSION THREAD

**Discussion by yDRvuiiGupz4** [jgraph/drawio-desktop#957] [Q&A]
Since updating to 17.4.2, the app crashes/quits every time I Create New Diagram.  Opening an existing diagram works as expected.   I went back to 17.2.1 and the crash is gone.  Windows 11 64-bit.

**Comment by davidjgraph** [comment]
We've had stability issues on Windows with recent electron builds, with a small number of users. Probably some of the issue is in our code, but we've not been able to reproduce the issues. If you're a VSCode user, you might consider the plugin there, https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio. That hasn't suffered from these issues.

**Comment by yDRvuiiGupz4** [comment]
Thanks for the quick follow-up.   I’ll skip the current release as 17.2.1 runs fine.  I’ll install your next release and see how it works.

Great app.

Jim

On Apr 15, 2022, at 9:44 AM, David Benson ***@***.***> wrote:


We've had stability issues on Windows with recent electron builds, with a small number of users. Probably some of the issue is in our code, but we've not been able to reproduce the issues. If you're a VSCode user, you might consider the plugin there, https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio <https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio>. That hasn't suffered from these issues.
